cli/state.rs

```diff
@@ -36,6 +28,9 @@ use std::sync::Arc;
 use std::thread::JoinHandle;
 use std::time::Instant;
 
+pub type RcState = Rc<State>;
```

Comment:
Maybe this alias should be renamed to `CliState`?

Reply:
Please no. The alias should be removed.

Reply:
It was actually pretty useful for porting - this is the type that is stored in the Gotham state - so you end up referencing it all the time.

Reply:
Removed